docs/sql-data-sources-jdbc.md

```diff
@@ -213,6 +213,14 @@ the following case-insensitive options:
   </tr>
 </table>
 
+It worth to mention that kerberos authentication with keytab is not always supported by the JDBC driver.<br>
```

Comment:
`It worth to mention that...` -> `Note that...` (cuz I see `Note that` statements in the other documents)

Reply:
Changed.